test/XRP/issued-currency-client-integration.test.ts

```diff
@@ -573,133 +589,110 @@ describe('IssuedCurrencyClient Integration Tests', function (): void {
-      issuer,
+      issuerWalletAuthTrustLines,
     )
 
     // WHEN the issuer unfreezes the trustline
     await issuedCurrencyClient.unfreezeTrustLine(
-      accountToUnfreeze.getAddress(),
+      walletMightHaveTrustLines.getAddress(),
       trustLineCurrency,
-      issuer,
+      issuerWalletAuthTrustLines,
     )
 
     const trustLines = await issuedCurrencyClient.getTrustLines(
-      issuer.getAddress(),
+      issuerWalletAuthTrustLines.getAddress(),
     )
 
-    const [unfrozenTrustLine] = trustLines
+    let unfrozenTrustLine
+    trustLines.forEach((trustLine) => {
+      if (trustLine.currency === trustLineCurrency) {
+        unfrozenTrustLine = trustLine
+      }
+    })
 
     // THEN the trust line is not frozen.
     assert.equal(unfrozenTrustLine.freeze, false)
     assert.equal(unfrozenTrustLine.limit, '0')
   })
 
-  it('disableRipplingForTrustLine - sets no rippling on trust line', async function (): Promise<
+  it('disableRipplingForTrustLine/enableRipplingForTrustLine', async function (): Promise<
     void
   > {
     this.timeout(timeoutMs)
-    const issuer = await XRPTestUtils.randomWalletFromFaucet()
-    const trustLinePeerAccount = await XRPTestUtils.randomWalletFromFaucet()
-
     // GIVEN an existing issuer account who has a trust line with a counter-party
-    await issuedCurrencyClient.requireAuthorizedTrustlines(issuer)
-
-    const trustLineCurrency = 'USD'
+    const trustLineCurrency = 'NRP'
     await issuedCurrencyClient.authorizeTrustLine(
-      trustLinePeerAccount.getAddress(),
+      walletMightHaveTrustLines.getAddress(),
       trustLineCurrency,
-      issuer,
+      issuerWalletAuthTrustLines,
     )
 
     const trustLineAmount = '1'
 
     // WHEN the issuer sets no rippling on the trust line
     await issuedCurrencyClient.disableRipplingForTrustLine(
-      trustLinePeerAccount.getAddress(),
+      walletMightHaveTrustLines.getAddress(),
       trustLineCurrency,
       trustLineAmount,
-      issuer,
+      issuerWalletAuthTrustLines,
     )
 
-    const trustLines = await issuedCurrencyClient.getTrustLines(
-      issuer.getAddress(),
+    let trustLines = await issuedCurrencyClient.getTrustLines(
+      issuerWalletAuthTrustLines.getAddress(),
     )
 
-    const [trustLine] = trustLines
+    let noRippleTrustLine
+    trustLines.forEach((trustLine) => {
+      if (trustLine.currency === trustLineCurrency) {
+        noRippleTrustLine = trustLine
+      }
+    })
 
     // THEN the trust line has noRipple enabled.
-    assert.equal(trustLine.noRipple, true)
-    assert.equal(trustLine.limit, trustLineAmount)
-  })
-
-  it('enableRipplingForTrustLine - clears noRipple on a trust line', async function (): Promise<
-    void
-  > {
-    this.timeout(timeoutMs)
-    const issuer = await XRPTestUtils.randomWalletFromFaucet()
-    const trustLinePeerAccount = await XRPTestUtils.randomWalletFromFaucet()
-
-    // GIVEN an existing issuer account who has a trust line with a counter-party,
```

Comment:
Why was this test deleted?

Reply:
ah it's because I combined it with the enableNoRippling test, much like you created combined cases for the flags tests, just to speed things up a little bit.